packages/whats-new/package.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
Seems like this will introduce changes to `yarn.lock`. Let's make sure to add them to the PR.